covid19/utils.py

```diff
@@ -11,21 +10,19 @@
 DATETIME_FORMAT = "%m-%d-%Y"
 
 
-def file_to_list(file: str, column: str) -> list:
+def file_to_list(file: str) -> list:
     """Convert a column from a file into a list of values
 
     Arguments:
         file {str} -- file name
-        column {str} -- column to convert into a list of values
 
     Raises:
-        Exception: If no values in column specified then file is considered empty
+        Exception: If no values in file passed then file is considered empty
 
     Returns:
         list -- A list with all values from the column given
     """
-    file_to_df = pd.read_csv(file, usecols=[column])
-    list_of_values = file_to_df[column].tolist()
+    list_of_values = pd.read_csv(file, squeeze=True).tolist()
```

Comment:
what does squeeze do?

Reply:
It converts a DataFrame into a series (single column). The last version of this function I was creating the series with the usecols=[column] argument and then converting into a list on the next line. This method removes the extra argument `column` and condenses into a single line.